Dockerfile

```diff
@@ -14,6 +14,8 @@ COPY pkg/ pkg/
 COPY proto/ proto/
 COPY Makefile Makefile
 
+# As (generated) proto/service.pb.go is _also_ committed, void the need to install protoc / protoc-gen-go plugin
+RUN touch proto/service.pb.go
```

Comment:
can you explain this change a bit more? I don't quite follow, we already copy the whole proto directory into the build container as per line 14

Reply:
sure, but depending on the `mtime` timestamp from the copy, it _may_ trigger rebuilding `proto/service.pb.go` which _may_ endup producing a different file from the corresponding `Makefile` target (it _shouldn't_, but ...) -- ie I want to be 100% sure we're using the committed file, also not enter into the biz of `go get ...`ing the protoc compiler at docker build time.